packages/communication-react/CHANGELOG.beta.md

```diff
@@ -4,91 +4,56 @@ This log was last generated on Wed, 04 Oct 2023 17:17:11 GMT and should not be m
-- Updating documentation to point to latest version of chat sdk ([PR #3589](https://github.com/azure/communication-ui-library/pull/3589) by joshlai@microsoft.com)
-- Improve ComplianceBanner preview in storybook ([PR #3635](https://github.com/azure/communication-ui-library/pull/3635) by 79475487+mgamis-msft@users.noreply.github.com)
-- Fix internal storybook build error in captions setting modal ([PR #3554](https://github.com/azure/communication-ui-library/pull/3554) by 2684369+JamesBurnside@users.noreply.github.com)
-- Updated documentation for file sharing ([PR #3581](https://github.com/azure/communication-ui-library/pull/3581) by 109105353+jpeng-ms@users.noreply.github.com)
-- Add ChatTheme for ImageGallery colors ([PR #3506](https://github.com/azure/communication-ui-library/pull/3506) by 107075081+Leah-Xia-Microsoft@users.noreply.github.com)
-- Improve storybook readability with tick quotes ([PR #3550](https://github.com/azure/communication-ui-library/pull/3550) by 77021369+jimchou-dev@users.noreply.github.com)
-- Update localized strings ([PR #3542](https://github.com/azure/communication-ui-library/pull/3542) by 79475487+mgamis-msft@users.noreply.github.com)
-- Change log grooming for 1.8.0 ([PR #3596](https://github.com/azure/communication-ui-library/pull/3596) by 79475487+mgamis-msft@users.noreply.github.com)
-- Updating storybook anchors to use ref ([PR #3625](https://github.com/azure/communication-ui-library/pull/3625) by joshlai@microsoft.com)
+- Add capabilitiesChanged event to CallWithChatAdapter ([PR #3570](https://github.com/azure/communication-ui-library/pull/3570) by 79475487+mgamis-msft@users.noreply.github.com)
 - Remove Fluent NorthStar ([PR #3578](https://github.com/azure/communication-ui-library/pull/3578) by 3941071+emlynmac@users.noreply.github.com)
 - Treeshake locales from composites ([PR #3615](https://github.com/azure/communication-ui-library/pull/3615) by 97124699+prabhjot-msft@users.noreply.github.com)
-- updating calling sdk version to 1.16.3 ([PR #3508](https://github.com/azure/communication-ui-library/pull/3508) by alkwa@microsoft.com)
-- Update `@azure/communication-calling` beta verison to 1.17.1-beta.5 ([PR #3552](https://github.com/azure/communication-ui-library/pull/3552) by 2684369+JamesBurnside@users.noreply.github.com)
+- Update localized strings ([PR #3542](https://github.com/azure/communication-ui-library/pull/3542) by 79475487+mgamis-msft@users.noreply.github.com)
+- Update `@azure/communication-signaling` version to 1.0.0-beta.20 ([PR #3524](https://github.com/azure/communication-ui-library/pull/3524) by 98852890+vhuseinova-msft@users.noreply.github.com)
+- Update `@azure/communication-calling` version to 1.16.3 and the beta verison to 1.17.1-beta.5 [PR #3552](https://github.com/azure/communication-ui-library/pull/3552) [PR #3508](https://github.com/azure/communication-ui-library/pull/3508)
```

Comment:
`[PR #3508](https://github.com/azure/communication-ui-library/pull/3508)` I don't think you need this entry in here since it only effects our stable build.